virtwho/main.py

```diff
@@ -161,7 +161,8 @@ def main():
             exit(1, err)
         # In order to keep compatibility with older releases of virt-who,
         # fallback to using libvirt as default virt backend
-        logger.info("No configurations found, using libvirt as backend")
+        logger.info("No configurations found (are there any '.conf' files in /etc/virt-who.d?), "
```

Comment:
Would it be useful to differentiate directly between no files in virt-who.d/ and having files that don't end in .conf?